libs/@blockprotocol/graph/src/types/subgraph/edges/outward-edge.ts

```diff
@@ -1,19 +1,29 @@
 import { EntityId, isEntityRecordId } from "../../entity.js";
 import { isOntologyTypeRecordId } from "../../ontology.js";
-import { Timestamp } from "../../temporal-versioning.js";
+import {
+  LimitedTemporalBound,
+  TemporalBound,
+  TimeInterval,
+  Timestamp,
+} from "../../temporal-versioning.js";
 import {
   isKnowledgeGraphEdgeKind,
   isOntologyEdgeKind,
   isSharedEdgeKind,
 } from "./kind.js";
-import { KnowledgeGraphOutwardEdge } from "./variants/knowledge";
-import { OntologyOutwardEdge } from "./variants/ontology";
+import { KnowledgeGraphOutwardEdge } from "./variants/knowledge.js";
+import { OntologyOutwardEdge } from "./variants/ontology.js";
 
 export type EntityIdAndTimestamp = {
   baseId: EntityId;
   timestamp: Timestamp;
 };
 
+export type EntityValidInterval = {
+  entityId: EntityId;
+  validInterval: TimeInterval<LimitedTemporalBound, TemporalBound>;
+};
```

Comment:
It refers to a period in an entity's existence whereby an edge was active, yes. It could be more than that I guess. In _some_ ways this implies that this entity was valid for at least this range, and it also indirectly implies that the other endpoint (not captured in this object) was valid for at least this range too.